lntest/harness.go

```diff
@@ -72,8 +76,8 @@ type NetworkHarness struct {
 // TODO(roasbeef): add option to use golang's build library to a binary of the
 // current repo. This will save developers from having to manually `go install`
 // within the repo each time before changes
-func NewNetworkHarness(r *rpctest.Harness, b BackendConfig, lndBinary string) (
-	*NetworkHarness, error) {
+func NewNetworkHarness(r *rpctest.Harness, b BackendConfig, lndBinary string,
+	useEtcd bool) (*NetworkHarness, error) {
```

Comment:
What about adding this instead as an argument to `NewNode`? This would let the tests eventually be run against nodes with heterogenous database backends.

Reply:
With the separation of local/remote dbs I think it's now better to have it as a param to `NewNetworkHarness` as otherwise the param needs to be passed to each individual `NewNode` call in the tests, making it harder to parametrize. For our current itests at least.